libraries/replace_between.rb

```diff
@@ -0,0 +1,66 @@
+#
+# Copyright:: 2018 Sous Chefs
```

Comment:
```suggestion
# Copyright:: 2019 Sous Chefs
```

Reply:
We probably don't even need licence headers as they're the same all throughout this cookbook